lib/modules/manager/woodpecker/__fixtures__/.woodpecker.v1.yml

```diff
@@ -0,0 +1,24 @@
+steps:
```

Comment:
please reduce file, we don't need so many deps for such a simple feature.

better also inline the fixture